bin/hatop

```diff
@@ -1162,9 +1165,10 @@ class Screen:
                 self.addstr(ypos, 1, 'HOTKEYS:',
                         curses.A_BOLD | curses.A_REVERSE)
                 self.addstr(ypos, 11,
+                        'F1=ENABLE_ALL_BACK  F2=DISABLE_ALL_BACK  '
```

Comment:
Let's relabel these "ENABLE" and "DISABLE", since this is the default (and usually correct) behavior.